### PR TITLE
SDK-1273: Improved docker files

### PR DESCRIPTION
**Pull request by vol4onok**
## Issue ticket link
Ticket: https://spryker.atlassian.net/browse/SDK-1273.

## Checklist before requesting a review
- [ ] I have added ticket link to the PR description.
- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

**Review by pavelmaksimov25 — DISMISSED** [review]

**Review by sergeyspryker — DISMISSED** [review]

File: infrastructure/sdk.Dockerfile [hunk 1]
```diff
@@ -20,35 +21,37 @@ FROM application-production-dependencies AS application-production-codebase
 
 RUN chown spryker:spryker ${srcRoot}
 
+ARG USER_UID
+RUN if [[ ! -z "${USER_UID}" ]]; then \
```
Comment (sergeyspryker) [inline infrastructure/sdk.Dockerfile:25]: Need to move it into local and debug files

File: bin/spryker-sdk.sh [hunk 2]
```diff
@@ -84,11 +84,23 @@ rest-api-start*)
     ;;
 esac
 
+install_composer () {
+    if [[ ! -d "$SDK_DIR/vendor" ]]; then
+       composer install --no-scripts --no-interaction --optimize-autoloader --ignore-platform-reqs;
```
[the remark this answers is not in the document]
Comment (vol4onok) [reply bin/spryker-sdk.sh:89]: it doesn't work locally. depends on local env.

[the remark this answers is not in the document]
Comment (pavelmaksimov25) [reply bin/spryker-sdk.sh:89]: ok, let it be then. So far sdk is not stable

**Review by pavelmaksimov25 — APPROVED** [review]

File: bin/spryker-sdk.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (vol4onok) [reply bin/spryker-sdk.sh:89]: exactly is not a problem. we just ignore everything we have locally and use versions from lock file.

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**